projects/steering/test/test_control_stalk.c

```diff
@@ -0,0 +1,114 @@
+#include "adc.h"
+#include "adc_periodic_reader.h"
+#include "can.h"
+#include "can_msg_defs.h"
+#include "event_queue.h"
+#include "exported_enums.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt_def.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "steering_can.h"
+#include "steering_control_stalk.h"
+#include "steering_digital_input.h"
+#include "steering_events.h"
+#include "test_helpers.h"
+
+#define STEERING_CAN_DEVICE_ID 0x1
+#define STEERING_CONTROL_STALK_LEFT_SIGNAL_VOLTAGE 1000
+#define STEERING_CONTROL_STALK_RIGHT_SIGNAL_VOLTAGE 2000
+#define STEERING_CC_INCREASE_SPEED_VOLTAGE 3000
+#define STEERING_CC_DECREASE_SPEED_VOLTAGE 4000
+#define STEERING_CC_BRAKE_PRESSED_VOLTAGE 5000
+#define INVALID_VOLTAGE 6000
+#define VOLTAGE_TOLERANCE_MV 100
+#define TIMER_INTERVAL_MS 50

```

Comment:
could you make this any lower for the test? we run all test cases whenever we commit to a PR, so it would save time, even if it's just a little